### DISCUSSION TITLE
Normal Installation Package?

### DISCUSSION THREAD

**Discussion by decayedxistence** [eoyilmaz/displaycal-py3#378] [General]
I know a lot of people that aren't code or tech savy are having an insane amount of trouble installing the Pyth3 version of DisplayCal. It is super confusing for many. Are there any future plans to release a regular installer like any other app of program for Mac and Windows?

Thanks so much for all of your hard work!

**Comment by jdavidbuerk** [comment]
I'm also here impatiently waiting a standard DMG or PKG installer file - I've been checking this fork for updates for months.  If it can be compiled, I don't understand why an installer can't be shared here; that said, I'm a pro photographer / videographer / (basic) graphic design peep, not a coder / programmer - I do a lot editing and running a business, and don't have time to learn Python compiling to do one single task - I just need software that works, and I don't trust other utility's profiles they've created - I'm currently running a display profile that was made with DisplayCal when it was still working natively on macOS, but it's old enough that my monitor has definitely shifted and needs to be recalibrated.

Any estimate when a DMG / PKG / similar will be released?

**Reply by MStraeten** [reply]
why not use the https://pypi.org/project/DisplayCAL/ approach? You don’t need to be a coder to type that into a terminal window…
a dmg will be releases after someone coded a GitHub runner to build that - but in FOSS no one is forced to do this in a given timeline …
